Engine/source/environment/scatterSky.cpp

```diff
@@ -85,8 +85,6 @@ const F32 ScatterSky::smViewerHeight = 1.0f;
 GFXImplementVertexFormat( ScatterSkyVertex )
 {
    addElement( "POSITION", GFXDeclType_Float3 );
-   addElement( "NORMAL", GFXDeclType_Float3 );
-   addElement( "COLOR", GFXDeclType_Color );
 }
```

Comment:
Good catch. that bein the case, why not kill it entirely, given GFXImplementVertexFormat( GFXVertexP ) :
https://github.com/GarageGames/Torque3D/blob/2044b2691e1a29fa65d1bdd163f0d834995433ce/Engine/source/gfx/gfxVertexTypes.cpp#L27